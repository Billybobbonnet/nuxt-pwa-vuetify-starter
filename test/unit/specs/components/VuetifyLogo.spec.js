import { mount } from "@vue/test-utils";
import VuetifyLogo from "~/components/VuetifyLogo";

describe("Vuetify logo", () => {
  it("renders to a snapshot", () => {
    const wrapper = mount(VuetifyLogo);
    expect(wrapper.element).toMatchSnapshot();
  });
});
