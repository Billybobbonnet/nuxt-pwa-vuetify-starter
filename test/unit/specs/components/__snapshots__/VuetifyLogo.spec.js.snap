// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Vuetify logo renders to a snapshot 1`] = `
<img
  alt="Vuetify.js"
  class="vuetify-logo"
  src="/icon.png"
  width="200"
/>
`;
